index.html

```diff
@@ -11,10 +11,12 @@ <h1><i>Thinkment: Philosophy and Ideology</i></h1>
     	 <h2>Overview</h2>
     	 .nice {font-family: sans-serif;}
     	</style>
+        <h1><img src="https://mk0itemsssrcorgqb419.kinstacdn.com/wp-content/uploads/2019/03/iStock-165790610-1000x569.jpg"></h1>
```

Comment:
Don't set the image as a header. Remove the `<h1>` and `</h1>` tags around the images.